doc/src/learn/objects.md

```diff
@@ -10,7 +10,7 @@ The basic unit of storage in Sui is **object**. In contrast to many other blockc
 
 All Sui objects have the following metadata:
 * A 20 byte globally unique ID. An object ID is derived from the digest of the transaction that created the object and  from a counter encoding the number of IDs generated by the transaction.
-* An 8 byte unsigned integer *version* representing the number of transactions that have included this object as an output.
+* An 8 byte unsigned integer *version* representing the [Lamport clock](https://en.wikipedia.org/wiki/Lamport_timestamp) of all input objects of the latest transaction.
```

Comment:
Internally we call this the "Lamport object version", to avoid confusion between this concept and exposing a concept of time to Move packages, but here I think it might just make sense to specify what you can assume about it, rather than what it is based on:

```suggestion
* An 8 byte unsigned integer *version* which monotonically increases with every transaction that reads or writes it.
```